docs-2.0/reuse/source_connect-to-nebula-graph.md

```diff
@@ -1,82 +1,57 @@
-   ```
-
-   Parameters and descriptions are as follows.
-
-   | Parameter | Description |
-   | - | - |
-   | `-h` | Shows the help menu. |
-   | `-addr` | Sets the IP address of the graphd service. The default address is 127.0.0.1. |
-   | `-port` | Sets the port number of the graphd service. The default port number is 9669.<!-- If you have deployed Nebula Graph in a docker container but Nebula Console is working outside the container, check the [source port](2.deploy-nebula-graph-with-docker-compose.md/#check_the_nebula_graph_service_status_and_port) of any nebula-graphd process and use it for connection. -->|
-   | `-u/-user` | Sets the username of your Nebula Graph account. Before enabling authentication, you can use any existing username. The default username is `root`. |
-   | `-p/-password` | Sets the password of your Nebula Graph account. Before enabling authentication, you can use any characters as the password. |
-   | `-t/-timeout`  | Sets an integer-type timeout threshold of the connection. The unit is second. The default value is 120. |
-   | `-e/-eval` | Sets a string-type nGQL statement. The nGQL statement is executed once the connection succeeds. The connection stops after the result is returned. |
-   | `-f/-file` | Sets the path of an nGQL file. The nGQL statements in the file are executed once the connection succeeds. The result will be returned and the connection stops then. |
-
-You can find more details in the [Nebula Console Repository](https://github.com/vesoft-inc/nebula-console/tree/{{console.branch}}).
+  Parameter descriptions are as follows:
+  
+  | Parameter | Description |
+  | - | - |
+  | `-h/-help` | Shows the help menu. |
+  | `-addr/-address` | Sets the IP address of the Graph service. The default address is 127.0.0.1. If Nebula Graph is deployed on [Nebula Cloud](https://docs.nebula-graph.io/2.6.2/nebula-cloud/1.what-is-cloud/), you need to create a Private Link and set the IP address of the Private Endpoint as the parameter value. |
+  | `-P/-port` | Sets the port number of the graphd service. The default port number is 9669. |
+  | `-u/-user` | Sets the username of your Nebula Graph account. Before enabling authentication, you can use any existing username. The default username is `root`. |
+  | `-p/-password` | Sets the password of your Nebula Graph account. Before enabling authentication, you can use any characters as the password. |
+  | `-t/-timeout`  | Sets an integer-type timeout threshold of the connection. The unit is second. The default value is 120. |
+  | `-e/-eval` | Sets a string-type nGQL statement. The nGQL statement is executed once the connection succeeds. The connection stops after the result is returned. |
+  | `-f/-file` | Sets the path of an nGQL file. The nGQL statements in the file are executed once the connection succeeds. The result will be returned and the connection stops then. |
+  | `-enable_ssl` | Enable SSL encryption when connecting to Nebula Graph. |
```

Comment:
```suggestion
  | `-enable_ssl` | Enables SSL encryption when connecting to Nebula Graph. |
```

Reply:
is it better to unify the singular and plural of the first word?

if so, the following sentences can be fixed too.